cli/options.go

```diff
@@ -190,7 +190,11 @@ func WithOsEnv(o *ProjectOptions) error {
 // WithEnvFile set an alternate env file
 // deprecated - use WithEnvFiles
 func WithEnvFile(file string) ProjectOptionsFn {
-	return WithEnvFiles(file)
+	var files []string
+	if file != "" {
```

Comment:
LGTM from a backward compatibility point of view, but we should also update compose to pass []string, see https://github.com/docker/compose/pull/10284/files#diff-2922e5cbd75acccf0c12784852f77f143247e72a57719e883225f10f960acd40L132-R232